pluton/cluster.go

```diff
@@ -43,6 +47,7 @@ type Info struct {
 	KubeletTag      string // e.g. v1.5.3_coreos.0
 	Version         string // e.g. v1.5.3+coreos.0
 	UpstreamVersion string // e.g. v1.5.3
+	Options                // pipe through how test was configured
```

Comment:
One use I had of Options being separate from a test struct was that one could do for most tests:
```
var defaults = pluton.Options{
        SelfHostEtcd:    false,
        InitialMasters: 1,
        InitialWorkers:  1,
}

func init() {
        // main test suite run on every PR
        harness.Register(pluton.Test{
                Name:    "bootkube.smoke",
                Run:     smoke,
                Options: defaults,
        })
        harness.Register(pluton.Test{
                Name: "bootkube.selfetcd.smoke",
                Run:  smoke,
                Options: pluton.Options{
                        SelfHostEtcd:   true,
                        InitialMasters: 1,
                        InitialWorkers: 1,
                },
        })
}
```
Its not perfect, but its an easy way for the owner of the test repo to declare what their normal cluster setup looks like. Anything not default has to specify ALL options still of course.

I will drop piping through Options into the Info struct at least though since it has no uses yet and I don't remember any good use-cases I had in mind at the time.